collections-csharp/ConcurrentQueueInCSharp/ConcurrentQueueInCSharp.Console/Consumer.cs

```diff
@@ -0,0 +1,28 @@
+﻿using ConcurrentQueueInCSharp.Models;
+
+namespace ConcurrentQueueInCSharp
```

Comment:
File level namespace will reduce nesting

Reply:
To tell the truth, I'm not the biggest fan of file-level namespaces as I find it less readable than the classic approach.

Reply:
@MichalBrylKaminski to each his own, but personally I find the reduced nesting to be a big win. We remove the initial 4 space/tab indent without changing any code flow. The only major difference is a removal if a single pair of brackets.

Reply:
I agree @jshergal. It all depends on personal preferences unless the project specifies it as a code standard.

Reply:
@MichalBrylKaminski - From the Code Maze implementation code guidelines for authors

![image](https://github.com/CodeMazeBlog/CodeMazeGuides/assets/1489638/d8d0b21c-6811-4741-8761-f40fa3d574dd)

Reply:
Right, fair enough. I don't recall this point in the guidelines. I have applied the changes.